index.html

```diff
@@ -0,0 +1,158 @@
+    <div class="headings">
+        <h2 class="quote">"Good Food Nourishes Souls."</h2>
+        <h1 class="topic">SHAKY'S ANNUAL COOKOUT 2023</h1> 
+        </div>
+    
+    <div class="home-message">
+        <p class="welcome-message">
+            The sun is shining, the grill is sizzling, and the aroma of delicious food is in the air. It's a beautiful day for a cookout! 
+        </p>
+    </div>
+            <div class="time-location">
+                <h2 class="time">2023-02-23(THUR)~ 24(FRI)</h2>
+                <p class="location">@The carnivore grounds Nairobi Kenya</p>
+            </div>
+        </div>    
+    </section>
+   
+    <section class="main-program" id="main-program">
+        <div>
+            <p class="main-program-heading">Main program</p>
+            <div class="red-line1"></div>
+        </div>
+        <div class="program-cards">
+            <div class="card">
+                <div class="icon">
+                    <i class="far fa-thin fa-handshake"></i>
+                  </div>
+                <h3 class="program-card-entry">Introduction</h3>
+                <p class="program-card-desc">Let get to know ourselves and share some insights about out favorite quisines</p>
+            </div>
+            <div class="card">
+                <img class="icon" src="./images/icons8-savouring-delicious-food-face-50.png">
+                <h3 class="program-card-entry">Appetizers</h3>
+                <p class="program-card-desc">Our professional cooks will make some of their favorite appetizers and let the food connoisseurs be the judge.</p>
+              </div>
+              
+            <div class="card">
+                <img class="icon" src="./images/icons8-barbecue-50.png">
+                <h3 class="program-card-entry">Entree</h3>
+                <p class="program-card-desc">Make your choice enjoy our savoury stakes grilled carbs and lucious salads </p>
+            </div>
+            <div class="card">
+                <image class ="icon" src="./images/icons8-fries-32.png"></image>
+                <h3 class="program-card-entry">Dessert</h3>
+                <p class="program-card-desc">We love them sweet some times extreemely sweet. I most cases extreemly cold</p>
+            </div>
+            <div class="card">
+<img class ="icon" src="./images/icons8-star-30.png">
+                <h3 class="program-card-entry">Ratings</h3>
+                <p class="program-card-desc">We love you feedback remember to vote for you favorite dishes and always keep your critique postive</p>
+            </div>
+        </div>
+        <button class="btn register-button mobile">Register Now!</button>
+        <div class="see-whole"><a href="#">SEE WHOLE PROGRAM</a> </div>
+    </section>
+    
+<section class="featured-chefs" id="featured-chefs">
+    <h2 class="main-program-heading">Featured Chefs</h2>
+    <div class="red-line1"></div>
+  <ul class="featured-chefs-list" id="chefs">
```

Comment:
- [ ] please in the mobile version, the speakers are arranged vertically but with each having its image on the left and info on the right.
yours
![Screenshot (812)](https://user-images.githubusercontent.com/65298048/219620283-eb5c5570-4c4f-4705-af0e-fb5e52ff94ec.png)
original
![Screenshot (813)](https://user-images.githubusercontent.com/65298048/219620471-c470b216-7d00-419a-a706-5155b4d09934.png)